src/data-browser/graphs/quarterly/index.jsx

```diff
@@ -1,33 +1,45 @@
-import React, { useState } from 'react'
-import { GraphsHeader } from './GraphsHeader'
-import { HomeLink } from '../../HomeLink'
-import { SectionFAQ } from './SectionFAQ.jsx'
-import { SectionFilerInfo } from './SectionFilerInfo'
-import { SectionGraphs } from './SectionGraphs'
-import { SectionSelector } from '../SectionSelector'
-import Error from '../../../common/Error'
-import '../graphs.css'
+import React, { useEffect, useState } from "react"
+import { GraphsHeader } from "./GraphsHeader"
+import { HomeLink } from "../../HomeLink"
+import { SectionFAQ } from "./SectionFAQ.jsx"
+import { SectionFilerInfo } from "./SectionFilerInfo"
+import { SectionGraphs } from "./SectionGraphs"
+import { SectionSelector } from "../SectionSelector"
+import { Switch, Route } from "react-router-dom"
+import Error from "../../../common/Error"
+import "../graphs.css"
 
-const SectionOptions = ['Graphs', 'Filer Info', 'FAQ']
+const SectionOptions = [
+  { label: "Graphs" }, // url for graphs comes from redux
+  { label: "Filers' Info", url: "/filers" },
```

Comment:
Sorry, I keep going back and forth on this, but maybe we should leave the label as `Filer Info` 😬 .  Even if `Filers' Info` is more accurate, it seems like `Filer Info` is just easier to read and reason about. Thoughts?